### PR TITLE
Updated version of hwid kit, added BroadcastReceiver to auto fill ver…

### PR DESCRIPTION
**Pull request by VladimirZhdanov**
Updated version of hwid kit, added BroadcastReceiver to auto fill verification number from sms.

### CODE REVIEW

File: app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java [hunk 1]
```diff
@@ -135,6 +155,45 @@ private void initView() {
...
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            Bundle bundle = intent.getExtras();
+
+            if (bundle != null && ReadSmsConstant.READ_SMS_BROADCAST_ACTION.equals(intent.getAction())) {
+                Status status = bundle.getParcelable(ReadSmsConstant.EXTRA_STATUS);
+                if (status.getStatusCode() == CommonStatusCodes.TIMEOUT) {
+
+                    // The service has timed out and no SMS message that meets the requirements is read. The service process ends.
+                    showToast("Time out");
```
Comment (javster) [inline app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java]: extract to strings

File: app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java [hunk 2]
```diff
@@ -135,6 +155,45 @@ private void initView() {
...
+                } else if (status.getStatusCode() == CommonStatusCodes.SUCCESS) {
+                    if (bundle.containsKey(ReadSmsConstant.EXTRA_SMS_MESSAGE)) {
+
+                        // An SMS message that meets the requirement is read. The service process ends.
+                        String result = bundle.getString(ReadSmsConstant.EXTRA_SMS_MESSAGE);
+                        String numberOnly= result.replaceAll("[^0-9]", "");
+                        showToast("Success: " + numberOnly);
```
Comment (javster) [inline app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java]: extract to strings

File: app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java [hunk 3]
```diff
@@ -135,6 +155,45 @@ private void initView() {
...
+                if (status.getStatusCode() == CommonStatusCodes.TIMEOUT) {
+
+                    // The service has timed out and no SMS message that meets the requirements is read. The service process ends.
+                    showToast("Time out");
+                } else if (status.getStatusCode() == CommonStatusCodes.SUCCESS) {
+                    if (bundle.containsKey(ReadSmsConstant.EXTRA_SMS_MESSAGE)) {
+
+                        // An SMS message that meets the requirement is read. The service process ends.
+                        String result = bundle.getString(ReadSmsConstant.EXTRA_SMS_MESSAGE);
+                        String numberOnly= result.replaceAll("[^0-9]", "");
```
Comment (javster) [inline app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java]: ctrl-alt-l

File: app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java [hunk 4]
```diff
@@ -135,6 +155,45 @@ private void initView() {
...
+
+                    // The service has timed out and no SMS message that meets the requirements is read. The service process ends.
+                    showToast(getString(R.string.time_out));
+                } else if (status.getStatusCode() == CommonStatusCodes.SUCCESS) {
+                    if (bundle.containsKey(ReadSmsConstant.EXTRA_SMS_MESSAGE)) {
+
+                        // An SMS message that meets the requirement is read. The service process ends.
+                        String result = bundle.getString(ReadSmsConstant.EXTRA_SMS_MESSAGE);
+                        String numberOnly = result.replaceAll("[^0-9]", "");
+                        showToast(getString(R.string.success) + numberOnly);
```
Comment (javster) [inline app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java]: В форматированную строку

File: app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java [hunk 5]
```diff
@@ -135,6 +155,45 @@ private void initView() {
...
+
+                    // The service has timed out and no SMS message that meets the requirements is read. The service process ends.
+                    showToast(getString(R.string.time_out));
+                } else if (status.getStatusCode() == CommonStatusCodes.SUCCESS) {
+                    if (bundle.containsKey(ReadSmsConstant.EXTRA_SMS_MESSAGE)) {
+
+                        // An SMS message that meets the requirement is read. The service process ends.
+                        String result = bundle.getString(ReadSmsConstant.EXTRA_SMS_MESSAGE);
+                        String numberOnly = result.replaceAll("[^0-9]", "");
+                        showToast(String.format("%s %s", getString(R.string.success), numberOnly));
```
Comment (ru-huawei-android) [inline app/src/main/java/com/huawei/hms/example/authservice/PhoneActivity.java]: <string name="ololo">Ololo %1$d</string>
getString(R.string.ololo, 1)

**Review by ru-huawei-android — APPROVED** [review]

**Pull request merged**